ignite/pkg/cosmosanalysis/cosmosanalysis_test.go

```diff
@@ -108,6 +108,24 @@ func (app *App) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.R
 func (app *App) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.ResponseEndBlock {
 	return app.mm.EndBlock(ctx, req)
 }
+`)
+	appFileSDKv47 = []byte(`
```

Comment:
For readability’s sake can you put this in another file. Additionally, can we have the whole app.go file generated by ignite v0.27 instead.